src/shared/components/case-editor/case-edit/case-edit.component.spec.ts

```diff
@@ -312,7 +312,7 @@ describe('CaseEditComponent', () => {
 
           expect(wizard.nextPage).toHaveBeenCalled();
           expect(routerStub.navigate).toHaveBeenCalled();
-          expect(component.form.get('data').get(CASE_FIELD_1.id)).toBeNull();
```

Comment:
we should add a separate unit test or 2 or more in this class with proper descriptions, for example:
`it should not clear hidden fields if readonly when leaving the page` or something like that

Reply:
Added couple of unit tests for "do not clear READONLY hidden field value" scenario for nextPage and previous page.